src/test/java/com/verint/todoappapi/ToDoIntegrationTest.java

```diff
@@ -0,0 +1,78 @@
+package com.verint.todoappapi;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.Before;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+import static io.restassured.RestAssured.given;
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.hasItems;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
Remove this. This is going to start up the application, we want to write the tests such that they are pointing at an already running environment. In which case the HOST and POST will need to be parameters to whatever is running the tests so it can be different between local and remote.